components/lvgl_esp32_drivers/lvgl_touch/tp_spi.c

```diff
@@ -51,9 +51,9 @@ void tp_spi_init(void)
 	};
 
 	spi_device_interface_config_t devcfg={
-		.clock_speed_hz=2*1000*1000,           //Clock out at 80 MHz
+		.clock_speed_hz=2*1000*1000,            //Clock out at 2 MHz
 		.mode=0,                                //SPI mode 0
-		.spics_io_num=-1,              //CS pin
+		.spics_io_num=TP_SPI_CS,                //CS pin
 		.queue_size=1,
 		.pre_cb=NULL,
 		.post_cb=NULL,
```

Comment:
According to [this](https://esp32.com/viewtopic.php?t=1499) post on the esp32 forum, setting a GPIO number in the `spics_io_num` field limits the number of devices up to three on the spi bus, it suggest to set `spics_io_num` to -1 and handle the cs control on the `pre_cb` and `post_cb` callbacks. Maybe we can try that so the number of slaves on the SPI bus isnt't limited to three. What do you think?

Reply:
That's interesting... unfortunately it looks like you have to modify the esp-idf NO_CS manifest constant in spi_master.c in order to make it work, and if you do that, then it seems like the esp-idf layer will handle more than three devices via the spics_io_num field.